### PR TITLE
Alps

### PR DESCRIPTION
**Pull request by valoeghese**
Alps variant in the Alpine Biome

### CODE REVIEW

File: src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java [hunk 1]
```diff
@@ -409,7 +426,11 @@ public static void init() {
 		OverworldBiomes.addContinentalBiome(TEMPERATE_RAINFOREST, OverworldClimate.COOL, 1.0);
 		OverworldBiomes.addContinentalBiome(SNOWY_RAINFOREST, OverworldClimate.SNOWY, 2.0);
 		OverworldBiomes.addContinentalBiome(ALPINE, OverworldClimate.SNOWY, 1.0);
-
+		
+		OverworldBiomes.addHillsBiome(ALPINE, ALPINE, 1);
```
Comment (valoeghese) [inline src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java]: This is to make the alps less common as hills

File: src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java [hunk 2]
```diff
@@ -381,6 +382,22 @@ public static void init() {
 				.addSpawnEntry(new Biome.SpawnEntry(EntityType.SALMON, 15, 3, 6))
 				.addSpawnEntry(new Biome.SpawnEntry(EntityType.COD, 15, 1, 5))
 				.build());
+		
+		ALPS = register("alps", TerrestriaBiome.builder()
+				.configuredSurfaceBuilder(SurfaceBuilder.DEFAULT, TerrestriaSurfaces.ALPS_CONFIG)
+				.precipitation(Biome.Precipitation.SNOW).category(Biome.Category.ICY)
+				.depth(3.7F)
+				.scale(0.3F)
+				.temperature(0.9F)
```
Comment (coderbot16) [inline src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java]: This is almost certainly incorrect... should be made much colder!

File: src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java (hunk 1 above)
Comment (coderbot16) [reply src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java]: Got it, makes sense.

**Comment by coderbot16** [comment]
Also, you forgot to add a language file entry

**Comment by coderbot16** [comment]
Merging this to fix it up myself, in the future you can just create branches here instead of having your own repository since I gave you write access

**Pull request merged**